### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Dani Tiago Faria dos Santos and Nicolas Duprat

### DISCUSSION THREAD

**Discussion by Dansnts** [heig-vd-dai-course/heig-vd-dai-course#463] [Show and tell]
**Pocket Monster - Octogone Edition**  : a console-based Pokémon battle simulator where players take on the role of a Trainer. 
The game allows Trainers to assemble a team of Pokémon, engage in battles against the computer, and manage their Pokémon collection through a simple text file database. 

**Key Features:**
- Trainers can choose from a variety of Pokémon to create their battle team. 
- Trainers can add new Pokémon and their moves to a persistent database stored in a simple text file (.txt).
- The console interface shows real-time updates during battles, including:
   - Each Pokémon's current HP.
   - Battle outcomes, including damage dealt and remaining health after each attack.

**Comment by GeraudSilvestri** [comment]
It is good for me. Be wary that you're project seems complicated and may require a lot of time implementing.

So reduce to a minimum the base features of your game and once those are working, iterate on it

**Comment by Dansnts** [comment]
Link to our repo : [Link](https://github.com/Dansnts/DAI-Labo02)


Last commit : [Last commit](https://github.com/Dansnts/DAI-Labo02/commit/2d53cd2f1b0f04e1f51f4842fad72df0784e6b23)
